CMakeLists.txt

```diff
@@ -62,6 +62,8 @@ endif()
 target_include_directories(docopt PUBLIC $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}> $<INSTALL_INTERFACE:include/docopt>)
 target_include_directories(docopt_s PUBLIC $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}> $<INSTALL_INTERFACE:include/docopt>)
 
+include(cmake/WarningsAsErrors.cmake)
```

Comment:
This is way more than fixing the project name.  I'm suspicious that -Werror is the right thing to include as part of a cmake cleanup pull request. I'd want to see CI tests passing after this change before accepting this change.

Reply:
what is the problem with this way to be correct while building this lib?

Reply:
There are no warnings and too no errors, luckily.